src/transformers/models/convnext/convert_convnext_to_pytorch.py

```diff
@@ -149,6 +150,31 @@ def convert_convnext_checkpoint(checkpoint_url, pytorch_dump_folder_path):
     image_processor = ConvNextImageProcessor(size=size)
     pixel_values = image_processor(images=prepare_img(), return_tensors="pt").pixel_values
 
+    # preprocess image
+    transformations = transforms.Compose(
+        [
+            transforms.Resize(256, interpolation=transforms.InterpolationMode.BICUBIC),
+            transforms.CenterCrop(224),
+            transforms.ToTensor(),
+            transforms.Normalize(
+                mean=IMAGENET_DEFAULT_MEAN,  # these are RGB mean+std values
+                std=IMAGENET_DEFAULT_STD,  # across a large photo dataset.
+            ),
+        ]
+    )
+
+    original_pixel_values = transformations(pixel_values).unsqueeze(0)  # insert batch dimension
+
+    processor = BitImageProcessor(
```

Comment:
We already have `ConvNextImageProcessor` should it would be great to leverage that class. `BitImageProcessor` is used for other models like BiT, DINOv2